addons/loyalty/models/loyalty_program.py

```diff
@@ -22,7 +22,10 @@ class LoyaltyProgram(models.Model):
         readonly=False, required=True, store=True, precompute=True)
     currency_symbol = fields.Char(related='currency_id.symbol')
     pricelist_ids = fields.Many2many(
-        'product.pricelist', string="Pricelist", domain="[('currency_id', '=', currency_id)]"
+        'product.pricelist',
+        string="Pricelist",
+        domain="[('currency_id', '=', currency_id)]",
+        help="This program is specific to this pricelist set.",
```

Comment:
Please sort field attributes according to our guidelines (Knowledge > Coding Style conventions > Sort field attributes section)